region/maridia/inner-yellow.json

```diff
@@ -1228,17 +1322,89 @@
                       "h_canCrouchJumpDownGrab"
                     ]}
                   ],
-                  "reusableRoomwideNotable": "Bug Sand Room Naked",
+                  "reusableRoomwideNotable": "Bug Sand Hole Naked",
+                  "note": [
+                    "Avoid the Yapping Maw after entering by not standing in one place for too long.",
+                    "Escape by wall jumping when the water is low or using a crouch jump and down grab when it's high.",
+                    "Kill the Zoa and quickly jump across the sand.",
+                    "Hitting the Zoas will drop Samus deep into the sand below."
+                  ]
+                },
+                {
+                  "name": "Naked Bug Sand Hole Damage Boost (R to L)",
+                  "notable": true,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "canTrickyJump",
+                    {"or":[
+                      "canWalljump",
+                      "h_canCrouchJumpDownGrab"
+                    ]},
+                    "canDamageBoost",
+                    {"enemyDamage": {
+                      "enemy": "Zoa",
+                      "type": "contact",
+                      "hits": 1
+                    }}
+                  ],
+                  "note": [
```

Comment:
If this is notable, can we represent it with a reusable strat?

Reply:
It looks like my plan was to share the notable with `Bug Sand Hole Naked`.  But the strats are done differently so I'll make another reusable for damage boosting.